src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -4,17 +4,50 @@
 
 public class StorageImpl<K, V> implements Storage<K, V> {
 
+    private static final int CAPACITY = 10;
+    private V[] valuesStorage;
+    private K[] keysStorage;
+    private int index;
+
+    public StorageImpl() {
+        valuesStorage = (V[]) new Object[CAPACITY];
+        keysStorage = (K[]) new Object[CAPACITY];
+        index = 0;
+    }
+
     @Override
     public void put(K key, V value) {
+        int indexOfKey = getIndex(key);
+        if (indexOfKey >= 0) {
+            valuesStorage[indexOfKey] = value;
+        } else {
+            valuesStorage[index] = value;
+            keysStorage[index] = key;
+            index++;
+        }
     }
 
     @Override
     public V get(K key) {
-        return null;
+        int indexOfKey = getIndex(key);
+        if (indexOfKey >= 0) {
+            return valuesStorage[indexOfKey];
+        } else {
+            return null;
+        }
```

Comment:
```suggestion
        } 
       return null;
```
we don't need else after return in if